drivers/wifi/Kconfig

```diff
@@ -35,5 +35,10 @@ config WIFI_OFFLOAD
 
 source "drivers/wifi/winc1500/Kconfig.winc1500"
 source "drivers/wifi/simplelink/Kconfig.simplelink"
+source "drivers/wifi/eswifi/Kconfig.eswifi"
+
+module = WIFI_DRIVER
```

Comment:
What is the purpose of this log level here as there is already log level for wifi few lines above?

Reply:
yes, you're right, fixed

Reply:
yes, you're right, fixed